KsApi/models/DeletePaymentMethodEnvelopeTests.swift

```diff
@@ -13,13 +13,15 @@ class DeletePaymentMethodEnvelopeTests: XCTestCase {
                   "expirationDate": "2055-11-01",
                   "id": "1057",
                   "lastFour": "1111",
-                  "type": "VISA"
+                  "type": "VISA",
+                  "stripeCardId": "pm_1OtGFX4VvJ2PtfhK3Gp00SWK",
```

Comment:
I erred on adding this field everywhere that references `storedCards`, even if the test didn't break without the field. I'd rather the data was accurate to what we'll get back from GraphQL.